collections/guides/collect_charge_cards_js/index.md

```diff
@@ -92,17 +92,19 @@ To set up a Reactor, head over to our Portal and set up a new Stripe Reactor. If
   </span>
 </span>
 
-Once you’ve created your Stripe Reactor, use the <code>reactor_id</code> and your Atomic Card token's ID to exchange for a Stripe token, which you'll be able to use to charge your customer.
+Once you’ve created your Stripe Reactor, use the <code>reactor_id</code> and your Atomic Card token's ID to exchange for a Stripe token, which you'll be able to use to charge your customer. 
 
 ```js
-const reactionToken = await bt.atomicCards.react(atomic_card.id, {
-    reactorId: REACTOR_ID
+const reactionToken = await bt.reactors.react(REACTOR_ID, {
```

Comment:
We should change this variable to something like `reactorResponse` - this is not a token anymore and the name could be confusing